.docker/wiremock/cms/mappings/dummy-479e26d6-5197-4d8f-a2dd-fefe64aabc14.json

```diff
@@ -7,7 +7,7 @@
   },
   "response" : {
     "status" : 200,
-    "body" : "[\n  {\n    \"category\": {\n      \"title\": \"Open\"\n    },\n    \"date\": \"2024-03-14\",\n    \"start_time\": \"09:00\",\n    \"end_time\": \"17:00\"\n  },\n    {\n    \"category\": {\n      \"title\": \"Open\"\n    },\n    \"date\": \"2024-03-15\",\n    \"start_time\": \"09:00\",\n    \"end_time\": \"17:00\"\n  }\n]",
+    "body" : "[\n  {\n    \"category\": {\n      \"title\": \"Open\"\n    },\n    \"date\": \"2024-03-22\",\n    \"start_time\": \"09:00\",\n    \"end_time\": \"17:00\"\n  },\n    {\n    \"category\": {\n      \"title\": \"Open\"\n    },\n    \"date\": \"2024-03-23\",\n    \"start_time\": \"09:00\",\n    \"end_time\": \"17:00\"\n  }\n]",
```

Comment:
Just a note about the naming of this file:
I guess it is temporary that the file is called dummy-* 
If it is going to be used in the future setup please remeber to call it something meaningful :)